src/_includes/docs/swift-package-manager/migrate-ios-project-manually.md

```diff
@@ -0,0 +1,76 @@
+Once you enable the Swift Package Manager feature, the Flutter CLI will try
+to migrate your project to add Swift Package Manager integration. This makes
+your project download the Swift packages that your Flutter plugins depend on.
+
+However, the Flutter CLI may be unable to migrate your project if you've
+modified it significantly.
+
+Please [file a bug][] before migrating manually to help the Flutter team improve
+the automatic migration.
+Please include the error message you received and consider including a copy of
+the of the following files in your bug report:
+
+* `ios/Runner.xcodeproj/project.pbxproj`
+* `ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme`
+   (or the xcsheme for the flavor used)
+
+### Step 1: Add FlutterGeneratedPluginSwiftPackage Package Dependency {:.no_toc}
+
+1. Open your app (`ios/Runner.xcworkspace`) in Xcode.
+2. Navigate to Package Dependencies for the project.
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/package-dependencies.png",
+   caption:"The project's package dependencies" %}
+
+3. Click the `+` button.
+4. Click the `Add Local...` button on the bottom of the dialog that opens.
+5. Navigate to `ios/Flutter/ephemeral/Packages/FlutterGeneratedPluginSwiftPackage`
+   and click the `Add Package` button.
+6. Ensure it is added to the `Runner` target and click the `Add Package` button.
```

Comment:
```suggestion
6. Ensure that it's added to the **Runner** target and click the `Add Package` button.
```

(The general rule of thumb, is that menu items should be bolded.)

Reply:
Fixed!